.babelrc

```diff
@@ -1,9 +1,12 @@
 {
-  "presets": ["@babel/preset-env"],
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-typescript"
+  ],
   "plugins": [
     "@babel/plugin-proposal-object-rest-spread",
     "@babel/plugin-transform-runtime",
     "@babel/plugin-proposal-export-default-from",
     "ramda"
   ]
-}
+}
```

Comment:
enable support of editorconfig in your ide to make it automatically [insert a final newline](https://github.com/aeternity/aepp-sdk-js/blob/e295ec90fe4bbac30451ddbc40f82e6a3b6e71e4/.editorconfig#L8)